modules/ensemble/lib/widget/youtube/youtube.dart

```diff
@@ -100,7 +101,14 @@ class YouTubeState extends WidgetState<YouTube> with YouTubeMethods {
             showControls: playerController.showControls,
             showFullscreenButton: playerController.showFullScreenButton,
             showVideoAnnotations: playerController.showVideoAnnotation));
-    super.initState();
+    WidgetsBinding.instance.addObserver(this);
+  }
+
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) {
+    if (state == AppLifecycleState.paused) {
```

Comment:
I am not sure how this solves the problem at all. The problem is that when you go from one screen to anther, the youtube video keeps playing. App is NOT being paused. How will this work?